src/Module.cpp

```diff
@@ -294,7 +294,11 @@ namespace utility {
             return;
         }
 
+#if defined(_M_X64)
         auto peb = (PEB*)__readgsqword(0x60);
+#else
+        auto peb = (PEB*)__readfsdword(0x60);
```

Comment:
This offset is incorrect for the PEB on 32-bit windows. It should be `fs:0x30`.

Reply:
Thanks, didn't know that. Fixed.